com.librumreader.librum.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: com.librumreader.librum
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: librum
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --filesystem=host
+  - --device=dri
+modules:
+  - python3-wheel.yaml
+  - python3-setuptools_scm.yaml
+  - python3-scikit-build.yaml
+  - python3-swig.yaml
+  - python3-libclang.yaml
+  - name: librum
+    buildsystem: cmake-ninja
+    build-options:
+      - build-args:
+        - --share-network
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release 
+      - -DBUILD_TESTS=Off
+    sources:
+        - type: git
+          branch: main
```

Comment:
NEVER a git branch. this is why the linter is unhappy.
```
        "module-librum-bin-source-git-no-commit-or-tag",
        "module-librum-bin-source-git-branch"
```

Reply:
When I do not specify a branch, it tries getting it from master, which fails. Is there another fix for that then?

Reply:
Never mind, I have tried removing the `branch: main` and it seems to work and not complain about not finding master now.